src/AvValidator/match.js

```diff
@@ -1,5 +1,6 @@
 import { isEmpty } from './utils';
+import get from 'lodash/get';
 
 export default function validate(value, context, constraint = {}) {
-  return isEmpty(value) || value === context[constraint.value] || constraint.errorMessage || false;
+  return isEmpty(value) || value === get(context, constraint.value) || constraint.errorMessage || false;
```

Comment:
is || false really necessary here? 
if anything in the chain evaluates to false it will return that regardless, right?